mcp/mcptools/qiskit_mcp.py

```diff
@@ -0,0 +1,230 @@
+from typing import Any
+import httpx
+from mcp.server.fastmcp import FastMCP
+#from qiskit_ibm_catalog import QiskitFunctionsCatalog
```

Comment:
remove

Reply:
This is necessary for the real qiskit function call. I'll think about how to mock the qiskit function and about this line.